server/src/middleware/eventHandlerWrapper.ts

```diff
@@ -1,10 +1,12 @@
 import { EventHandlerWrapper } from 'typings';
+import { log } from '../logger/opts';
 
 const eventHandlerWrapper: EventHandlerWrapper = (ctx, handler) => (payload, callback) => {
   try {
     handler(ctx, payload, callback);
   } catch (ex) {
-    console.log('Error w  chuj: ', ex);
+    ctx.io.to(ctx.socket.id).emit('EXCEPTION', ex);
+    log.info(ex);
```

Comment:
tak jak w przypadku routa restowego, jeżeli tu się posypało coś to raczej to jest error a nie info